keyboards/kprepublic/bm60hsrgb_ec/readme.md

```diff
@@ -0,0 +1,13 @@
+# BM60HSRGB_EC
+
+A 60% hotswap inswitch RGB keyboard with rotary encoder from KP Republic. 
+
+* Keyboard Maintainer: [luisgit9](https://github.com/luisgit9)
+* Hardware Supported: BM60HSRGB_EC/rev2
+* Hardware Availability: [KP Republic](https://kprepublic.com/products/bm60ec-bm60-ec-rgb-60-gh60-hot-swappable-custom-mechanical-keyboard-pcb-programmed-qmk-via-rgb-switch-type-c-rotary-knob)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Added instructions to the keyboard readme.md, as well as on the rev2 readme.md.